src/makeCalculator.js

```diff
@@ -4,7 +4,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(a) {
+      return this.result + a;
```

Comment:
[CODE STYLE] - Instead of returning the addition result directly, use an assignment operator with `+=` to add to the existing value of `this.result`, as per the checklist.